### ISSUE TITLE
FR: Add 'sh' to container image to support aws-sdk-go cred setup

### ISSUE THREAD

**Issue by allisaurus** [awslabs/amazon-ecs-local-container-endpoints#30]
'sh' executable is required by aws-sdk-go to invoke credential_process.

See [related PR](https://github.com/awslabs/amazon-ecs-local-container-endpoints/pull/29) for more context.

**Comment by samuelkarp** [comment]
One approach here might be modifying the SDK to remove the call to `sh`.  It's not super clear what specifically the SDK is trying to support; I don't know if we really expect folks to write a full shell command in their `~/.aws/config` or not.

**Comment by geckofu** [comment]
@samuelkarp Hi, the SDK is trying to support [external credentials sourcing](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-sourcing-external.html). We at [ThePlant](http://github.com/theplant) write shell commands in `~/.aws/config` because we use OIDC federation to manage IAM access. This is the tool we built for this purpose: [oidc2aws](https://github.com/theplant/oidc2aws). The config file looks like this:

```
# Get developer credentials via employee's Google account
[profile developer]
credential_process = oidc2aws arn:aws:iam::562055475000:role/developer
region = ap-northeast-1
```
---
BTW, maybe I can change the base image from `alpine` to `amazonlinux` in this [PR](https://github.com/awslabs/amazon-ecs-local-container-endpoints/pull/29)?

**Comment by allisaurus** [comment]
Hey @geckofu - apologies for the delay in getting back to you! Yes, if you want to update https://github.com/awslabs/amazon-ecs-local-container-endpoints/pull/29 to use the `amazonlinux` base image we'll give it anther look. Thanks again for taking the time to contribute and for clarifying your use case.

**Comment by geckofu** [comment]
Thanks @allisaurus I have changed the base image to `amazonlinux` in #31 (couldn't update the original PR in place). Happy to help

**Comment by PettitWesley** [comment]
Released in 1.1.0
